tidb-cloud/migrate-from-oracle-using-aws-dms.md

```diff
@@ -0,0 +1,152 @@
+
+## Step 2. Create an Oracle DB instance
+
+Create an Oracle DB instance in the VPC you just created, and remember the password and grant it public access. You must enable public access to use the AWS Schema Conversion Tool. Note that granting public access in the production environment is not recommended.
+
+For instructions about how to create an Oracle DB instance, see [Creating an Oracle DB instance and connecting to a database on an Oracle DB instance](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_GettingStarted.CreatingConnecting.Oracle.html).
+
+![Create Oracle RDS](/media/tidb-cloud/aws-dms-from-oracle-to-tidb-2.png)
+
+## Step 3. Prepare the table data in Oracle
+
+Insert some data into the Oracle DB instance. You can use the github event dataset and download it from [GH Archive](https://gharchive.org/). It contains 10000 rows of data. Use the following SQL script to execute it in Oracle.
+
+- [table_schema_oracle.sql](https://github.com/pingcap-inc/tidb-integration-script/blob/main/aws-dms/oracle_table_schema.sql)
+- [oracle_data.sql](https://github.com/pingcap-inc/tidb-integration-script/blob/main/aws-dms/oracle_data.sql)
+
+After you finish executing the SQL script, check the data in Oracle. The following example uses [DBeaver](https://dbeaver.io/) to query the data:
+
+![Oracle RDS Data](/media/tidb-cloud/aws-dms-from-oracle-to-tidb-3.png)
+
+## Step 4. Create a TiDB Cloud Serverless Tier cluster
+
+1. Log in to the [TiDB Cloud console](https://tidbcloud.com/console/clusters). Navigate to the **Clusters** page for your project.
+
+2. Create a free Serverless Tier cluster.
+
+    ![Create TiDB Cloud Serverless Tier](/media/tidb-cloud/aws-dms-from-oracle-to-tidb-4.png)
+
+3. After the cluster is created, click **Security Settings** to set the user password.
+
+    ![Set TiDB Cloud Serverless Tier Password](/media/tidb-cloud/aws-dms-from-oracle-to-tidb-5.png)
+
+    ![Set TiDB Cloud Serverless Tier Password](/media/tidb-cloud/aws-dms-from-oracle-to-tidb-6.png)
+
+4. Click the **Generate** button generate the password. Note down the generated password. Click the **Submit** button.
+
+5. Click the **Connect** button to connect your TiDB cluster:
+
+    ![Connect to TiDB Cloud Serverless Tier](/media/tidb-cloud/aws-dms-from-oracle-to-tidb-7.png)
+
+## Step 5. Create an AWS DMS replication instance
+
+1. Go to the [Replication instances](https://console.aws.amazon.com/dms/v2/home#replicationInstances) page in the AWS DMS console, and switch to the corresponding region.
+
+2. Create an AWS DMS replication instance with `dms.t3.large` in the VPC.
+
+    ![Create AWS DMS Instance](/media/tidb-cloud/aws-dms-from-oracle-to-tidb-8.png)
+
+## Step 6. Create DMS endpoints
+
+1. In the [AWS DMS console](https://console.aws.amazon.com/dms/v2/home), click the replication instance that you just created.
+
+2. Create the Oracle source endpoint and the TiDB target endpoint.
```

Comment:
Creating oracle source endpoint and target TiDB is a bit confusing for me and here is why

     -  In step 1 you want the user to go to the DMS Console and Click the replication instance. Is this necessary I think the menu item on the left has the  navigation item that allows you to create end point isnt it ? I will send you a screen shot on Lark.

Reply:
This is unnecessary, I'll change it.